### DISCUSSION TITLE
preview and cleaner scripts not called as long as external preview open

### DISCUSSION THREAD

**Discussion by bugsbugsbux** [gokcehan/lf#2433] [Q&A]
`lf --version # 28`

hi, i want to use swayimg as my image previewer, and wrote a little scirpt to do so which positions the window it opens above the preview area. this works fine, but blocks the preview and cleaner scripts from running until it is closed. why?

I tried to start swayimg as a background process and disown it, and i am marking the preview as volatile by exiting the previewer with code 1 after starting swayimg. if at least one, the previewer or the cleaner ran, i can close the window if that's what prevents the other script from running...

**Comment by jlesquembre** [comment]
@herrvonvoid could you share your script? I'm trying to do the same, but I don't get the right position

**Comment by bugsbugsbux** [comment]
https://gist.github.com/herrvonvoid/84fb34c0752f3f457fb360f3ba2d3c11

**Comment by CatsDeservePets** [comment]
I will convert this into a discussion.

Also, the gist link is now the following:
https://gist.github.com/bugsbugsbux/84fb34c0752f3f457fb360f3ba2d3c11
